src/test/ui/consts/issue-77062-large-zst-array.rs

```diff
@@ -1,4 +1,4 @@
-// build-pass
```

Comment:
that should stay as `build-pass`. We also have to successfully deal with this in llvm